modules/terraform/output.go

```diff
@@ -49,6 +50,48 @@ func OutputRequiredE(t *testing.T, options *Options, key string) (string, error)
 	return out, nil
 }
 
+// OutputList calls terraform output for the given variable and returns its value as a list.
+// If the output value is not a list type, then it fails the test.
+func OutputList(t *testing.T, options *Options, key string) []string {
+	out, err := OutputListE(t, options, key)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return out
+}
+
+// OutputListE calls terraform output for the given variable and returns its value as a list.
+// If the output value is not a list type, then it returns an error.
+func OutputListE(t *testing.T, options *Options, key string) ([]string, error) {
+	out, err := RunTerraformCommandE(t, options, "output", "-no-color", "-json", key)
+	if err != nil {
+		return nil, err
+	}
+
+	outputMap := map[string]interface{}{}
+	if err := json.Unmarshal([]byte(out), &outputMap); err != nil {
+		return nil, err
+	}
+
+	value, containsValue := outputMap["value"]
+	if !containsValue {
+		return nil, fmt.Errorf("Output doesn't contain a value for the key %q", key)
+	}
+
+	var list []string
+	switch t := value.(type) {
+	case []interface{}:
+		list = make([]string, len(t))
```

Comment:
Since these lists are short, you may as well declare `list` above via `list := []string{}`.